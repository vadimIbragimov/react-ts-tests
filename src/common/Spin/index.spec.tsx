import React from 'react';
import { mount, render, shallow } from 'enzyme';
import { Spin } from './index';

describe('BlankDiv component', () => {
  it('should slhallow BlankDiv component', () => {
    const component = shallow(<Spin />);
    expect(component).toMatchSnapshot();
  });
  it('should mount BlankDiv component', () => {
    const component = mount(<Spin />);
    expect(component).toMatchSnapshot();
  });
  it('should render BlankDiv component', () => {
    const component = render(<Spin />);
    expect(component).toMatchSnapshot();
  });
});
