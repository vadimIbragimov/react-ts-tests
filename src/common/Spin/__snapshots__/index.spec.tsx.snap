// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BlankDiv component should mount BlankDiv component 1`] = `
<Spin>
  <Spin
    size="default"
    spinning={true}
    wrapperClassName=""
  >
    <div
      className="ant-spin ant-spin-spinning"
    >
      <span
        className="ant-spin-dot ant-spin-dot-spin"
      >
        <i
          className="ant-spin-dot-item"
        />
        <i
          className="ant-spin-dot-item"
        />
        <i
          className="ant-spin-dot-item"
        />
        <i
          className="ant-spin-dot-item"
        />
      </span>
    </div>
  </Spin>
</Spin>
`;

exports[`BlankDiv component should render BlankDiv component 1`] = `
<div
  class="ant-spin ant-spin-spinning"
>
  <span
    class="ant-spin-dot ant-spin-dot-spin"
  >
    <i
      class="ant-spin-dot-item"
    />
    <i
      class="ant-spin-dot-item"
    />
    <i
      class="ant-spin-dot-item"
    />
    <i
      class="ant-spin-dot-item"
    />
  </span>
</div>
`;

exports[`BlankDiv component should slhallow BlankDiv component 1`] = `
<Spin
  size="default"
  spinning={true}
  wrapperClassName=""
/>
`;
